packages/api/tests/mock.factories.js

```diff
@@ -2112,6 +2112,42 @@ function fakeNotification(defaultData = {}) {
   };
 }
 
+async function populateDefaultRevenueTypes() {
+  const types = [{ revenue_name: 'Crop Sale', revenue_translation_key: 'CROP_SALE' }];
```

Comment:
When I set `revenue_type_id: 1`, I got the index error.
https://github.com/LiteFarmOrg/LiteFarm/actions/runs/6178207876/job/16771222403/#step:6:1725
The latest code works in beta but not in local. (I couldn't find a solution that would work for both env) After LF-3595 is complete, I will update the middleware and tests, and this should no longer be a problem...